CMake/itkCompilerChecks.cmake

```diff
@@ -10,10 +10,10 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND
   message(FATAL_ERROR "LLVM Clang 3.4 or later is required.")
 endif ()
 
-# Minimum compiler version check: Apple Clang >= 5.1 (Xcode 5.1)
+# Minimum compiler version check: Apple Clang >= 7.0.2 (Xcode 7.2.1)
```

Comment:
The [release notes](https://developer.apple.com/library/archive/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html#//apple_ref/doc/uid/TP40001051-CH1-SW876), under sub-heading "Apple LLVM Compiler Version 6.1" say "This updated compiler includes full support for the C++14 language standard". So it may not be necessary to bump to version 7.

Reply:
@dzenanz Strictly speaking I think you're right, in the sense that some Apple Clang versions before 7.0.2 may already support C++14. But I understood from Sean (@seanm) that it should be sufficient for the latest ITK version to support Apple Clang >= 7.0.2.

Note that back in December 2019 ("before corona"), https://github.com/InsightSoftwareConsortium/ITK/blob/333bfc690dca71a63ea884b0ac42eaa63e62e2c3/Documentation/SupportedCompilers.md already said so:

> AppleClang 7.0.2 and later (from Xcode 7.2.1) should be supported

OK?